### ISSUE TITLE
Failed to load Plugin ODIN

### ISSUE THREAD

**Issue by VCasecnikovs** [memgraph/odin#9]
Hi,
I am no MacOS m1.
I did all according to instructions and using docker-compose.

When I try to enable ODIN it writes Failed to load Plugin ODIN

**Comment by katarinasupe** [comment]
Hi @VCasecnikovs, did you clone the repository inside the plugins folder (your_vault/.obsidian/plugins)?
Can you tell me which exact steps did you take?

**Comment by VCasecnikovs** [comment]
Yes, I have cloned it in the folder, have added .env inside the ODIN's folder.
Have run docker compose.
And were trying to turn on plugin using Obsidian interface in the settings.
When I do the last step, it does not turnig on and there is a pop-up:
"Failed to load Plugin ODIN"

**Comment by katarinasupe** [comment]
TBH not sure why that's not working as expected. Can you try manual installation?

**Comment by service-fabric-dev** [comment]
Sorry to revive an old thread here, but I get a similar behavior. Opening the Obsidian console, I see the following:
![image](https://github.com/user-attachments/assets/e9b17838-db77-464e-afca-2ee4095b76b9)
The docker hosting is a hard requirement for my team to adopt this tool, but I can attempt the manual installation if it will help with debugging this issue.

**Comment by EmailToXjk** [comment]
Windows11, same question.
